recipes/jupyterlab-classic/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "jupyterlab-classic" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "94e83b6fbfcf0e3ad195883ea8794a7ac2135ca7177d6a9e0ebd60ddc21a175a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
you may wish to try `--no-deps`, if the pip solver is the issue...

Reply:
no-deps should be default in our builds, no need to specify it.

Reply:
good to know!